types.d.ts

```diff
@@ -1262,28 +1262,37 @@ export interface InputMediaDocument {
   disable_content_type_detection?: Boolean;
 }
 
-/** This object represents the contents of a file to be uploaded. Must be posted using multipart/form-data in the usual way that files are uploaded via the browser. */
+/**
+ * This object represents the contents of a file to be uploaded.
+ * Must be posted using multipart/form-data in the usual way that files are uploaded via the browser.
+ * @deprecated
+ */
 export type InputFile =
   | FileId
   | InputFileByPath
   | InputFileByReadableStream
   | InputFileByBuffer
   | InputFileByURL;
 
+/** @deprecated */
```

Comment:
We have breaking changes in minor releases all the time. This is inevitable if the Telegram Bot API constantly makes breaking changes. If we would try to follow semver, more or less every release would be major, that does not make sense. (Btw the same reasons why typescript itself it not following it.)

I know that this is a bit radical and very uncommon for “regular” libraries, but in this specific case, I would simply remove the types instead of deprecating them first just to remove them in the subsequent major release.

Reply:
What would you do with (non-generic) `Telegram`, `TelegramP` and `Opts`, which depend on `InputFile`? I'd leave them as-is, perhaps deprecate them.

Reply:
I'd leave them, too. They become aliases that point to the corresponding properties in DefaultTypegram, confer https://github.com/KnorpelSenf/typegram/blob/f1ef6a0a4e0c92d4b3943dc8e2e874b7ebf12cb2/alias.d.ts#L9

Reply:
How can you prevent them from being accidentally re-exported while leaving them in?

Reply:
I have to correct myself, I would leave the `InputMedia*` types in, not the `InputFile*` types. I would move the latter to `telegraf`. They do not belong here.

Reply:
Fair enough. I'd make this a major release.